hoomd/md/CMakeLists.txt

```diff
@@ -143,6 +143,9 @@ set(_md_headers ActiveForceComputeGPU.h
                 PotentialBondGPU.h
                 PotentialBondGPU.cuh
                 PotentialBond.h
+                PotentialMeshBondGPU.h
+                PotentialMeshBondGPU.cuh
+                PotentialMeshBond.h
```

Comment:
I used the name `PotentialMeshBond.ext` as I plan to add more potentials that are applied to the mesh triangles rather than the mesh bonds (ie. bending forces or area conservation forces).